app/src/main/java/lt/vilnius/tvarkau/activity/ReportRegistrationActivity.kt

```diff
@@ -31,7 +31,7 @@ class ReportRegistrationActivity : BaseActivity() {
                 .beginTransaction()
                 .replace(android.R.id.content, PhotoInstructionsFragment.newInstance())
                 .addToBackStack(null)
-                .commit()
+                .commitAllowingStateLoss()
```

Comment:
Hmm, forgot to test that...
There was a crash when asking for camera permissions, after approval it tries to display photo instructions and crashes.